### ISSUE TITLE
[issue]:  Windows 7 (VHD image) unable to boot 

### ISSUE THREAD

**Issue by mikolaj612** [ventoy/Ventoy#2063]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.84

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

Legacy BIOS Mode

### Partition Style

MBR

### Disk Capacity

32

### Disk Manufacturer

Patriot

### Image file checksum (if applicable)

None

### What happened?

Hello.
I tried to boot VHD image of Windows 7 SP1 (clean - from MSDN), I also applied VHD plugin as mentioned in FAQ.
However it won't boot no matter what I do - it's gaving me same error everytime 0x00007b.

I made VHD image of Windows 10 and it's working fantastic all way long - for both images I used VirtualBOX 6.1.40 and stored them as a static image with VHD extension.

Any idea what's going on, or where I made a mistake? 
It would be great to have an option to boot Windows 7 VHD from time to time - same way I could do with Windows 10.

**Comment by nhtt2023** [comment]
Have you tried booting Windows 7 SP1.VHD from VirtualBOX or booting Windows 7 SP1.VHD directly from pc/laptop?

**Comment by mikolaj612** [comment]
> Have you tried booting Windows 7 SP1.VHD from VirtualBOX or booting Windows 7 SP1.VHD directly from pc/laptop?

My VHD image been made with VritualBOX so it's working fine when used within VM.
However it doesn't work with Ventoy VHD boot - when Windows 10 - made same way will work without any issues.

I do not know how to overcome this.

**Comment by steve6375** [comment]
So are you using NTFS for partition 1
And have you changed the Ventoy vhd Img file plugin to the win7 one?

**Comment by mikolaj612** [comment]
> So are you using NTFS for partition 1 And have you changed the Ventoy vhd Img file plugin to the win7 one?

1st partition - when Ventoy is installed is FAT32 (there is no issue with Windows 10 as I mentioned), main partition where VHD are located and some other stuff is NTFS.

I used plugin from GitHub to boot VHD - it's working for Windows 10, however Win7 based one gave BSOD, Win10 version do not work with Windows 7 too.

**Comment by steve6375** [comment]
Win 7 legacy 32bit or 64 bit?

**Comment by mikolaj612** [comment]
> Win 7 legacy 32bit or 64 bit?

x86 (32 bit), same goes for Windows 10.

**Comment by steve6375** [comment]
Not sure plugin works with 32bit?

**Comment by mikolaj612** [comment]
> Not sure plugin works with 32bit?

When it came to Windows 10 it's working great.

**Comment by steve6375** [comment]
But that is a different plugin

**Comment by mikolaj612** [comment]
> But that is a different plugin

Yup, but no one stated that plugins are working only for x64 version.

**Comment by steve6375** [comment]
No one stated you were using win7 32-bit either 😑

**Comment by mikolaj612** [comment]
> No one stated you were using win7 32-bit either 😑

Actually my bad - you're right.
I hope that something could be done - if not I will stay with Windows 10 x86.

**Comment by nhtt2023** [comment]
Can you give me your download link for Windows 7 SP1.vhd?

**Comment by vrbfvrbf** [comment]
What version of Win7 are you using (Home, Pro, Ultimate)? Only Win7 Ultimate supports VHD booting.

**Comment by MaksKub** [comment]
Hi, I have the same problem!😔 
I have Windows 7 Ultimate x64 
ventoy_vhd boot.img plugin for windows 7 
It gives either just a black screen (almost always and everything freezes, only turning off the computer completely helps) or errors : 0xc000000e ; (File: \boot\memtest.exe )0xc0000225 
The procedure of actions:
Creating a vhd.
VHD connection and win7ultimate_x64.iso to VirtualBox. 
Launching VirtualBox and installing the system on a vhd.
Transferring vhd to usb Ventoy.
Booting from a usb Ventoy.
Selecting win7ultimate.vhd.
And a black screen appears and the system freezes.😔😔
